homeassistant/helpers/entity_registry.py

```diff
@@ -292,6 +292,14 @@ def async_clear_config_entry(self, config_entry):
     return await task
 
 
+@callback
+def async_entries_for_device(registry: EntityRegistry, device_id: str) \
+        -> List[RegistryEntry]:
+    """Return entries that match a device."""
+    return [entity for entity in registry.entities.values()
```

Comment:
This returns a list of entries, so use `entry` as variable name instead of `entity`.